swagger/swagger.yaml

```diff
@@ -0,0 +1,106 @@
+swagger: "2.0"
+info:
+  title: API for Fabric8 Analytics HPF Insights
+  description: Spec for the REST API of HPF insights model.
+  version: "0.1"
+basePath: "/api/v1/"
+consumes:
+  - application/json
+produces:
+  - application/json
+paths:
+  /liveness:
+    get:
+      tags: [Service settings]
+      operationId: src.flask_endpoint.liveness
+      summary: Get job service liveness
+      responses:
+        200:
+          description: Service is alive
+  /readiness:
+    get:
+      tags: [Service settings]
+      operationId: src.flask_endpoint.readiness
+      summary: Get job service readiness
+      responses:
+        200:
+          description: Service is ready
+  /companion_recommendation:
+    post:
+      tags: [Recommendation API]
+      operationId: src.flask_endpoint.hpf_scoring
+      description: Return companion package recommendations for a user stack
+      summary: Takes an array list of ecosystem + package_list as input, and provides recommendation for each package list. For ecosystems that don't match the loaded model's ecosystem, an error message is returned.
+      parameters:
+        - in: body
+          name: body
+          required: true
+          schema:
+            $ref: '#/definitions/UserStack'
+      responses:
+        '200':
+          schema:
+            $ref: '#/definitions/SuccessResponse'
+          description: Companion recommendations along with their associated probabilities
+        '202':
+          schema:
+            $ref: '#/definitions/FailureResponse'
+          description: No ecosystem provided to load the flask app
```

Comment:
@sara-02, What I can understand from the description, the status code should be some 4XX as it is client side error. 202 is for `Accepted` calls, which might mean that client sent the correct information but we couldn't return the result due to some known reasons( for eg, the epv couldn't be analysed due to model failure maybe, but the data was kept by server(maybe in queue) and once model was back to normal, it was then processed).